public/app/features/alerting/unified/components/silences/MatchedSilencedRules.tsx

```diff
@@ -96,13 +96,7 @@ function useColumns(): MatchedRulesTableColumnProps[] {
       id: 'created',
       label: 'Created',
       renderCell: function renderSummary({ data: { matchedInstance } }) {
-        return (
-          <>
-            {matchedInstance.activeAt.startsWith('0001')
-              ? '-'
-              : matchedInstance.activeAt.substr(0, 19).replace('T', ' ')}
-          </>
-        );
+        return <>{dateTime(matchedInstance.activeAt).format('YYYY-MM-DD HH:mm:ss')}</>;
```

Comment:
**issue:** I think we should still render `–` for dates that start with `0001` due to a bug we're still tracking down